### ISSUE TITLE
Error using string resource ad unit id 

### ISSUE THREAD

**Issue by brewin** [AppLovin/AppLovin-MAX-SDK-Android#144]
The ad unit id is correct and test ads show correctly, but errors are shown in logcat. I'm using a string resource because I have different ad unit ids for debug and release builds.

```
    <com.applovin.mediation.ads.MaxAdView xmlns:maxads="http://schemas.applovin.com/android/1.0"
        android:id="@+id/adView"
        android:layout_width="match_parent"
        android:layout_height="@dimen/adViewHeight"
        android:background="@color/black"
        maxads:adUnitId="@string/adUnitId" />
```
```
E/AppLovinSdk: [MediationService] Please double-check the ad unit @2131755205 for BANNER : java.lang.Throwable: 
        at com.applovin.impl.mediation.MediationServiceImpl.loadAd(Unknown Source:118)
        at com.applovin.impl.mediation.ads.MaxAdViewImpl$1.run(Unknown Source:251)
        at android.os.Handler.handleCallback(Handler.java:938)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loopOnce(Looper.java:201)
        at android.os.Looper.loop(Looper.java:288)
        at android.app.ActivityThread.main(ActivityThread.java:7839)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
```

**Comment by thomasmso** [comment]
Hi - unfortunately that's currently a limitation of the `MaxAdView`, could you hardcode the Ad Unit ID in the meantime?

**Comment by thomasmso** [comment]
We just released Android SDK v11.1.1 which addresses the issue. Thank you for your valuable feedback!
https://github.com/AppLovin/AppLovin-MAX-SDK-Android/releases/tag/release_11_1_1
